CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 
 ## 🛑 Breaking changes 🛑
 
+- Support `Snappy` and `Zstd` for confighttp.go (#4441)
```

Comment:
Please add a note for users to get some guidance about the configuration change needed to move the compression configuration out of the exporter config and into the http client config.

Reply:
added. Thank you for letting me know